ocs_ci/deployment/deployment.py

```diff
@@ -1732,9 +1734,12 @@ def deploy_acm_hub_unreleased(self):
         run_cmd(f"oc create -f {constants.ACM_HUB_UNRELEASED_ICSP_YAML}")
 
         logger.info("Writing tag data to snapshot.ver")
-        image_tag = config.ENV_DATA.get(
-            "acm_unreleased_image", config.ENV_DATA.get("default_acm_unreleased_image")
-        )
+        acm_version = config.ENV_DATA.get("acm_version")
+
+        image_tag = get_latest_acm_tag_unreleased(version=acm_version)
+        if not image_tag:
+            raise TagNotFoundException("Couldn't find given tag!")
```

Comment:
I would update the function `get_latest_acm_tag_unreleased` itself to raise an exception if it will not find a tag and document it in raises block in docstring.

Reply:
Done